processmanager/manager.go

```diff
@@ -163,6 +179,11 @@ func collectInterpreterMetrics(ctx context.Context, pm *ProcessManager,
 		summary[metrics.IDELFInfoCacheMiss] =
 			metrics.MetricValue(pm.elfInfoCacheMiss.Swap(0))
 
+		summary[metrics.IDTraceCacheHit] =
+			metrics.MetricValue(pm.frameCacheHit.Swap(0))
+		summary[metrics.IDTraceCacheMiss] =
+			metrics.MetricValue(pm.frameCacheMiss.Swap(0))
```

Comment:
This is reusing the `IDTraceCache*` even though its now repurposed as "frame cache". The cache is about the same thing - ebpf-to-decorated traces/frames. But the granularity changed from traces to frames. Does someone think the old ID should be obsoleted and and a new allocated for this? I assumed for now reusing the ID would be ok...